### PR TITLE
Adding method to check db exists and integrating it with the init db …

### PR DESCRIPTION
**Pull request by matdenkas**
…script

### CODE REVIEW

**Review by lovelysandlonelys — APPROVED** [review]
this makes sense to me!

**Comment by abeasley1722** [comment]
LGTM

**Review by abeasley1722 — APPROVED** [review]

File: tools/init_db.py [hunk 1]
```diff
@@ -31,16 +30,23 @@ def readInitCSV(csvFileName: str) -> list:
...
-    # Insert reference and mapping data
-    sqlorm.insert_ref_dataDatum(readInitCSV('dataDatum.csv'))
-    sqlorm.insert_ref_dataLocation(readInitCSV('dataLocation.csv'))
-    sqlorm.insert_ref_dataSeries(readInitCSV('dataSeries.csv'))
-    sqlorm.insert_ref_dataSource(readInitCSV('dataSource.csv'))
-    sqlorm.insert_ref_dataUnit(readInitCSV('dataUnit.csv'))
-    sqlorm.insert_data_mapping(readInitCSV('dataMapping.csv'))
+
+
+    dbEists = sqlorm.DB_exists()
```
Comment (bestrada33) [inline tools/init_db.py:35]: Guessing this should be dbExists? Missing the 'x'

File: tools/init_db.py [hunk 2]
```diff
@@ -31,16 +30,23 @@ def readInitCSV(csvFileName: str) -> list:
...
+    if dbEists == -1: # Clear out all data if some tables but not all tables are found
+        sqlorm.drop_DB()
+        dbEists = 0
+
+    if dbEists == 0: # Make the DB
+        sqlorm.create_DB()
+
+        # Insert reference and mapping data
+        sqlorm.insert_ref_dataDatum(readInitCSV('dataDatum.csv'))
```
Comment (bestrada33) [inline tools/init_db.py:44]: So, if the DB exists and has all tables it just overwrites the reference data no matter what? Even if the reference data was already in there?

**Review by bestrada33 — APPROVED** [review]
LGTM. Seems to work well. Only left one comment, but should be fine

**Pull request merged**